docs/usage/dns-search-path-optimization.md

```diff
@@ -0,0 +1,120 @@
+---
+title: DNS Search Path Optimization
+---
+
+# DNS Search Path Optimization
+
+## DNS Search Path
+
+Using fully qualified names has some downsides, e.g. it may become harder to move deployments from one landscape to the
```

Comment:
Thanks for clarifying. Maybe you can elaborate some of the points from the above comment in the doc and explain why "it may become harder to move deployments from one landscape to the next" (the downsides of hardcoding the FQDN or making it configurable per deployment/landscape).